### PR TITLE
updated the pac-resolver from 4.1.0 to 5.0.0

### LINKED ISSUE #373: `pac-resolver` vulnerability in package-lock.json
**Describe the bug**

This affects the package pac-resolver before 5.0.0. This can occur when used with untrusted input, due to unsafe PAC file handling. NOTE: The fix for this vulnerability is applied in the node-degenerator library, a dependency written by the same maintainer.

**Parent Issue**
- https://github.com/PalisadoesFoundation/talawa-api/issues/369

**References**
- https://github.com/advisories/GHSA-9j49-mfvp-vmhm

**Remediation**
The earliest fixed version is 5.0.0.

**Screenshots**

image

### PR DESCRIPTION
**Pull request by asmitsirohi**
**What kind of change does this PR introduce?**

Bug Fix

**Issue Number:**

Fixes #373 

**Did you add tests for your changes?**

No

**Snapshots/Videos:**

NA

**If relevant, did you update the documentation?**

NA

**Summary**

I have updated the package pac-resolver from 4.1.0 to 5.0.0 in package-lock.json.

**Does this PR introduce a breaking change?**

No

**Other information**

NA

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Review by palisadoes — CHANGES_REQUESTED** [review]
Your solution will probably not work. Let me know if you have you tested your solution. The GitHub alert we got below states that the fix will require updates to:

- `pm2`
- `pac-resolver`
- `pac-proxy-agent`

image

**Comment by asmitsirohi** [comment]
I have already updated pm2 version to 5.1.2 and pac-resolver version to 5.0.0

image

image

I verify the fix after running the npm audit command, as it is not showing any vulnerabilities for pac-resolver.

image

**Review by palisadoes — APPROVED** [review]

**Comment by palisadoes** [comment]
@asmitsirohi Thanks for the clarification! Let's get someone else to review.

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "mongoose-paginate-v2": "^1.3.18",
     "morgan": "^1.10.0",
     "netmask": ">=2.0.1",
-    "pm2": "^4.5.5",
+    "pm2": "^5.1.2",
```
Comment (sumitra19jha) [inline package.json:57]: Our work currently doesn't involve dealing with proxy configuration in-depth but Version above 5 is good to go as malicious requests on the local network to remotely run arbitrary code inside the Node.js process whenever an HTTP request is created is restricted.

**Review by sumitra19jha — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (asmitsirohi) [reply package.json:57]: I didn't get this, Can you please elaborate on what I need to do to resolve this one.

**Pull request merged**